lib/infra_conversion_throttler.rb

```diff
@@ -1,15 +1,24 @@
 class InfraConversionThrottler
   def self.start_conversions
     pending_conversion_jobs.each do |ems, jobs|
-      running = ems.conversion_hosts.inject(0) { |sum, ch| sum + ch.active_tasks.size }
+      running = ems.conversion_hosts.inject(0) { |sum, ch| sum + ch.active_tasks.count }
+      $log&.debug("There are currently #{running} conversion hosts running.")
```

Comment:
Why `$log&.debug`?  `$log` shouldn't be `nil`.  Also, you probably only want to log to debug if the logger is in debug mode, so something like this may be more appropriate..
```ruby
$log.debug { "There are currently #{running} conversion hosts running." }
```

Reply:
yes, please remove this from the PR